roles/serviceassurance/tasks/component_qdr.yml

```diff
@@ -45,6 +46,19 @@
           - expose: true
             http: true
             port: 8672
+        sslProfiles:
+          - caCert: {{ meta.name }}-interconnect-openstack-ca
+            credentials: {{ meta.name }}-interconnect-openstack-credentials
+            generateCaCert: true
+            generateCredentials: true
+            name: openstack
+          - caCert: {{ meta.name }}-interconnect-inter-router-ca
+            credentials: {{ meta.name }}-interconnect-inter-router-credentials
+            generateCaCert: true
+            generateCredentials: true
+            mutualAuth: true
```

Comment:
Oooh neat

Reply:
Yea once I figured that part out, I was like, "cooooool!"